packages/exports/src/zipDownloadedDriveFiles.ts

```diff
@@ -0,0 +1,33 @@
+import archiver from "archiver";
+import { downloadDriveFile } from "downloadDriveFile";
+import { PassThrough } from "stream";
+
+export const ext = ["pdf", "docx", "pptx"] as const;
+
+type Ext = typeof ext;
+
+export const zipDownloadedDriveFiles = async (
```

Comment:
Looks like this isn't imported anywhere and there's a duplicate definition elsewhere